packages/base/command-result.gts

```diff
@@ -222,6 +226,10 @@ class CommandResultEmbeddedView extends Component<typeof CommandResult> {
   </template>
 }
 
+export function add(value1: number, value2: number) {
+  return value1 + value2;
```

Comment:
Just a heads up - I'm adding this as a math helper in https://github.com/cardstack/boxel/pull/1521/files#diff-89ed62e9ed0e7ef8d23a884faf267c8d41ec9c66af01e6c313efb8df65faa511

Reply:
I removed data-test-result-card-idx and used ol so I have no need for the math-helper. Thanks for the heads up